test/unit/test_tpm2_auth_util.c

```diff
@@ -94,27 +94,77 @@ static void test_tpm2_password_util_from_optarg_str_escaped_hex_prefix(void **st
     tpm2_session_close(&session);
 }
 
+FILE mocked_file_stream;
+const char *mocked_file_data = "sekretpasswrd";
+
+FILE * __real_fopen(const char *path, const char *mode);
+FILE * __wrap_fopen(const char *path, const char *mode) {
+    if (strcmp (path, "test_tpm2_auth_util_foobar")) {
+        printf("REAL CALLED\n");
+        return __real_fopen(path, mode);
+    }
+    return mock_ptr_type(FILE*);
+}
+
+size_t __real_fread(void *ptr, size_t size, size_t nmemb, FILE *stream);
+size_t __wrap_fread(void *ptr, size_t size, size_t nmemb, FILE *stream) {
+    if (stream != &mocked_file_stream) {
+        return __real_fread(ptr,size,nmemb,stream);
+    }
+    UNUSED(stream);
+    memcpy(ptr, mocked_file_data, size * nmemb);
+    return mock_type (size_t);
+}
+
+long __real_ftell(FILE *stream);
+long __wrap_ftell(FILE *stream) {
+    if (stream != &mocked_file_stream) {
+        return __real_ftell(stream);
+    }
+    UNUSED(stream);
+    return mock_type (long);
```

Comment:
we don't do spaces between function name and opening parenthesis. This applies here and throughout